jestSetup.js

```diff
@@ -1 +1 @@
-jest.mock('./lib/storage/providers/SQLiteStorage', () => require('./lib/storage/providers/__mocks__/SQLiteStorage'));
+jest.mock('./lib/storage/NativeStorage', () => require('./lib/storage/__mocks__/NativeStorage'));
```

Comment:
Agree with all of the above.

>When desired (specific test suite) it can be put in place, while the default can be to always use the generic storage mock (after all we're don't have any other mocks right now but localforage mock)

Agree and we can always come back later to do it. And who knows, maybe one day we will see SQLite in the browser?